tests/xtro-sharpie/MacCatalyst-MapKit.ignore

```diff
@@ -0,0 +1,11 @@
+## defined twice for iOS (likely to make it available to OSX)
+### MKOverlayRenderer.h
+### MKOverlayView.h (based on UIView so not in OSX headers list)
+!missing-pinvoke! MKRoadWidthAtZoomScale is not bound
+
+
+## unsorted
+
+!missing-field! MKAnnotationCalloutInfoDidChangeNotification not bound
+!missing-protocol-conformance! MKUserLocation should conform to MKAnnotation
+!missing-protocol-member! MKOverlay::coordinate not found
```

Comment:
Why is this one ignored?

Reply:
I'm not sure, but I just copied the contents of the ios ignore file (It fails on xtro otherwise)